src/contracts/GPv2AllowanceManager.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.7.0;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+/// @title Gnosis Protocol v2 Allowance Manager Contract
+/// @author Gnosis Developers
+contract GPv2AllowanceManager {
+    using SafeERC20 for IERC20;
+
+    /// @dev A struct representing transfers to be executed as part of a batch
+    /// settlement.
+    struct Transfer {
+        address owner;
+        IERC20 token;
+        uint256 amount;
+    }
+
+    /// @dev The owner contract of the allowance manager. The owner is set at
+    /// creation time and cannot change.
+    address private immutable owner;
```

Comment:
namingnit: I'm not a huge fan of the term "Owner" in this context. Isn't it more an AllowanceTarget (or just target)?

Reply:
Maybe `recipient` so that it uses the same language as the OpenZeppelin ERC20 implementations?